docs/guide/localization/static.md

```diff
@@ -103,6 +104,53 @@ If you are customizing Apostrophe **user interface components**, the `$t()` meth
 As a reminder, the Vue.js components of the user interface are not connected to any Vue app you may be running for your website visitors. The registration method will not be automatically available outside the UI components.
 :::
 
+### Localizing with string interpolation
+
+Consider the following string: "*Displaying 8 articles*." Maybe in this case we are telling end users how many articles match a set of filters they have selected (e.g., published last month and only those about cooking). In this case we don't know the *exact* string to translate since the number will change. The variable part, the number, is also in the middle (for some languages, at least) so we could not even translate only the text part very easily. In cases like this we use **string interpolation**.
+
+String interpolation is a process of generating a text string that is partly dynamic. In the previous paragraph's example, the number of articles is dynamic, or variable, based on other input (the chosen filters).
+
+Regardless of whether we are localizing text in templates, server-side code, or UI Vue files, interpolation works essentially the same way. The string or localization key is still the first argument in the localization function. **Then an object is passed as a second argument to the l10n function, containing interpolation properties that match keys in curly braces.**
+
+Template example:
+
+<AposCodeBlock>
+  ```django
+    {{ __t('Displaying {{ count }} articles', {
```

Comment:
If we're going to use `count` we really should cover pluralization too. We're supporting i18next JSON version 3, so the syntax is that if a second key is available in the JSON file with a `_plural` suffix it is used when count is not equal to `1`, otherwise the exactly matching key is used. This applies **only** if the interpolated value is named `count`.

Reply:
That's valuable, but I don't want to complicate this section. I'll change the example.